demos/instance_segmentation_demo/python/instance_segmentation_demo.py

```diff
@@ -184,11 +183,12 @@ def main():
 
         render_start = time.time()
 
+        frames_processed += 1
```

Comment:
In some demos, object_detection for instance, frames counting in raw outputs starts with 0, in others, like here it starts with 1. I think it should always start with 0, like in cpp demos.

Reply:
Updated for `instance_segmentation_demo` and `text_spotting_demo`, now it starts with 0 frame for all demos.